src/views/tutorial-view/components/collection-card/types.ts

```diff
@@ -0,0 +1,11 @@
+import { Collection as ClientCollection } from 'lib/learn-client/types'
+
+export interface CollectionCardProps
+  extends Pick<
+    ClientCollection,
+    'id' | 'name' | 'slug' | 'theme' | 'description'
```

Comment:
Have you considered making all of these props one single prop (maybe named `collection`) that's object? One benefit would be less maintenance on the `CollectionCardProps` API, because the maintenance would move to the prop object if more display/logic attributes are needed from `ClientCollection`.

Reply:
I dig that! 💯 I like that it'll also prevent repetitious `thing={thing}` passing of props (without the dreaded `{...spread}`. Would it be alright if we handle that in the PR where we implement `CollectionCard` rather than this PR?

Reply:
That's totally good with me! 👍🏻